factcast-factus/src/main/java/org/factcast/factus/applier/DefaultEventApplier.java

```diff
@@ -310,6 +311,11 @@ private static boolean isEventHandlerMethod(Method m) {
                 createSingleParameterTransformer(m, null, type);
             }
 
+            // exclude MockitoMocks
```

Comment:
Why would you want to do that?

Reply:
Otherwise, it will complain that methods are duplicate.
The mechanics will find all methods of all non-static inner classes. Unfortunately, Mockito generates an inner class extending the outer (weird isn't it) and thus duplicating the method.